public/doc/openapi-3-v1.2.json

```diff
@@ -1505,72 +1505,37 @@
-                                "status": {
-                                    "type": "integer",
-                                    "example": 400
-                                },
-                                "details": {
-                                    "type": "string",
-                                    "example": "Bad Request"
-                                }
-                            }
-                        }
-                    }
-                }
-            },
-            "403ForbiddenResponse": {
-                "type": "object",
-                "description": "Caller is forbidden to perform the action.",
-                "properties": {
-                    "errors": {
-                        "type": "Array",
-                        "description": "Errors resulting from the API query",
-                        "items": {
-                            "type": "object",
-                            "properties": {
-                                "status": {
-                                    "type": "integer",
-                                    "example": 403
-                                },
-                                "details": {
-                                    "type": "string",
-                                    "example": "Forbidden Request"
-                                }
-                            }
+                            "$ref": "#/components/schemas/HttpApiError"
                         }
                     }
                 }
             },
-            "404NotFoundResponse": {
+            "HttpApiError": {
                 "type": "object",
-                "description": "The requested resource is not found.",
+                "description": "API Error",
                 "properties": {
-                    "errors": {
-                        "type": "Array",
-                        "description": "Errors resulting from the API query",
-                        "items": {
-                            "type": "object",
-                            "properties": {
-                                "status": {
-                                    "type": "integer",
-                                    "example": 404
-                                },
-                                "details": {
-                                    "type": "string",
-                                    "example": "Record not found"
-                                }
-                            }
-                        }
+                    "status": {
+                        "type": "string",
+                        "description": "HTTP status code",
+                        "example": "400"
+                    },
+                    "details": {
+                        "type": "string",
+                        "description": "Error details",
+                        "example": "Bad Request"
+                    },
+                    "code": {
```

Comment:
@hsong-rh Do we need code at this point?

Reply:
not really